<launch>
  <arg name="m" default="$(find sr_recognizer)data/models_2opt" doc="Models directory"  />
  <arg name="t" default="$(find sr_recognizer)data/photo1.pcd" doc="Test file"  />
  <arg name="con" default="$(find sr_recognizer)cfg/multipipeline_config.xml" doc="Recognizer config file"  />
  <arg name="arg" default="-z 2.5 -c 5" doc="Additional arguments as stated in the library. For help arg:=-h. "/>

  <node pkg="sr_recognizer" type="recognizer_server" name="recognizer_server" output="screen">
    <param name="models_dir" value="$(arg m)" />
    <param name="config" value="$(arg con)" />
    <param name="test_file" value="$(arg t)" />
    <param name="arg" value="$(arg arg)" />     
  </node>
</launch>
